pkg/oauthflow/oidc_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package oauthflow
+
+import (
+	"errors"
+	"github.com/coreos/go-oidc/v3/oidc"
+	"github.com/magiconair/properties/assert"
+	"reflect"
+	"testing"
+	"unsafe"
+)
+
+// reflect hack because "claims" field is unexported by oidc IDToken
```

Comment:
Nit: maybe link to the upstream issue about exporting those fields in case we can remove this once it gets closed?

Reply:
Added the link to the PR opened in the upstream project - https://github.com/coreos/go-oidc/pull/329